tests/test_warp.py

```diff
@@ -1268,3 +1268,31 @@ def test_reproject_dst_alpha(path_rgb_msk_byte_tif):
             dst_alpha=4)
 
         assert dst_arr[3].any()
+
+
+@pytest.mark.skipif(
+    (gdal_version.major == 2 and gdal_version.minor == 2),
```

Comment:
@Kirill888 I'm going to merge this PR and then fix the line above, post-merge, so that the test won't be skipped when we update GDAL on Travis-CI to 2.2.4.